docs/sphinx/user_guide/feature/iteration_spaces.rst

```diff
@@ -168,14 +174,14 @@ IndexSets
 --------------
 
 A ``RAJA::TypedIndexSet`` is a container that can hold an arbitrary collection
-of segment objects of arbitrary type as illustrated in the following figure.
+of segment objects of arbitrary type as illustrated in the following figure,
+where we have to contiguous ranges and an irregularly-spaced list of indices.
```

Comment:
Probably "_two_ contiguous ranges" instead of "to"?

Reply:
Fixed